app/models/event.rb

```diff
@@ -8,13 +9,13 @@ class Event < ActiveRecord::Base
 
   scope :sequenced, -> { where.not(sequence_id: nil).order(sequence_id: :asc) }
   scope :chronologically, -> { order(created_at: :asc) }
+  scope :excluding_sequence_ids, -> (ranges) { where RangeSet.to_ranges(ranges).map {|r| "(sequence_id NOT BETWEEN #{r.first} AND #{r.last})"}.join(' AND ') }
```

Comment:
Looks like we don't use this scope?

Reply:
I'd like to keep it for the future though, sentimental attachement. can I leave it commented for a while?